### PR TITLE
.coafile: Also lint docs/conf.py

### PR DESCRIPTION
**Pull request by userzimmermann**
Emerged from discussion https://github.com/coala/coala/commit/e275ebbf9bdad5f256dbc00c79887a88a000292c#commitcomment-22478720

Closes https://github.com/coala/coala/issues/4844

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -252,23 +253,23 @@
...
-#latex_show_urls = False
+# atex_show_urls = False
 
 # Documents to append as an appendix to all manuals.
-#latex_appendices = []
+# atex_appendices = []
 
 # If false, no module index is generated.
-#latex_domain_indices = True
+# atex_domain_indices = True
```
Comment (sils) [inline docs/conf.py]: removing the l?

Comment (shreyans800755) [inline docs/conf.py]: I hope this (removing `l`) is not caused by coala applyPatchAction. Is it ?

File: docs/conf.py [hunk 2]
```diff
@@ -296,14 +297,13 @@
...
-#texinfo_appendices = []
+# exinfo_appendices = []
 
 # If false, no module index is generated.
-#texinfo_domain_indices = True
+# exinfo_domain_indices = True
 
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
-#texinfo_show_urls = 'footnote'
+# exinfo_show_urls = 'footnote'
```
Comment (shreyans800755) [inline docs/conf.py]: All of these last 10 changes have lost first character and is being replaced by space.

Comment (jayvdb) [inline docs/conf.py]: ya, seems like it is an automated problem; maybe a bug in coala! or just a regex in a text editor

File: docs/conf.py (hunk 1 above)
Comment (userzimmermann) [inline docs/conf.py]: @shreyans800755 Luckily it isn't! It's just my fault ;)

File: docs/conf.py (hunk 2 above)
Comment (userzimmermann) [inline docs/conf.py]: @shreyans800755 @jayvdb @sils Exactly the latter suggestion... Just a little mistake in my replace regex :D Sorry... Will correct now!

**Comment by userzimmermann** [comment]
@sils @shreyans800755 Find/replace faux pas fixed ;) Please review, ack, merge :)

File: docs/conf.py [hunk 3]
```diff
@@ -45,12 +48,11 @@
 source_suffix = '.rst'
 
 # The encoding of source files.
-#source_encoding = 'utf-8-sig'
+# source_encoding = 'utf-8-sig'
 
 # The master toctree document.
 master_doc = 'index'
 
-from datetime import date
```
Comment (shreyans800755) [inline docs/conf.py]: I guess this is not part of linting. Can we have separate commit for this ?

File: docs/conf.py [hunk 4]
```diff
@@ -61,7 +63,6 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-from coalib.misc.Constants import VERSION
```
Comment (shreyans800755) [inline docs/conf.py]: Same as above.

**Comment by userzimmermann** [comment]
@shreyans800755 Commit message says: `And make necessary changes to docs/conf.py for passing` ;)

**Comment by shreyans800755** [comment]
Are these changes for passing or refactoring ?

**Comment by shreyans800755** [comment]
Passing it is then.
Please update the commit:
```
.coafile: Add file in the list of files to be linted
Update the file to follow code styles required by bears.
Closes <issue>
```

**Comment by userzimmermann** [comment]
@shreyans800755 C'mon. That advice is a bit too much ;)

**Comment by userzimmermann** [comment]
@shreyans800755 Let's not argue ;) I'm thankful that you took your time to review this again. I made a compromise change to the commit message

**Comment by shreyans800755** [comment]
ack 687ec1b

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by jayvdb** [comment]
This broke https://github.com/coala/coala/issues/3785

**Pull request merged**